odbc/src/handles/definitions.rs

```diff
@@ -47,6 +47,15 @@ impl MongoHandle {
         }
     }
 
+    pub fn statement_as_connection(&self) -> Option<&Connection> {
```

Comment:
Can you please rename this to something like "get_parent_connection" for clarity?
You are not casting the statement as a connection. The statement stashes a reference to the connection.